test/topology_experimental_raft/test_topology_upgrade.py

```diff
@@ -69,5 +72,10 @@ async def test_topology_upgrade_basic(request, manager: ManagerClient):
     logging.info("Checking consistency of data in system.topology and system.cdc_generations_v3")
     await check_system_topology_and_cdc_generations_v3_consistency(manager, hosts)
 
+    await wait_until_last_generation_is_in_use(cql)
+
+    logging.debug("Sleeping for 1 second to make sure there are writes to the CDC table in the last generation")
+    await asyncio.sleep(1)
+
```

Comment:
This isn't needed and only hardens the test, right? I don't see how some assert in `verify` could fail if no writes were sent to the last generation. This change is fine, I just want to make sure I understand.

Reply:
> This isn't needed and only hardens the test, right?

Yes, we want to test that writes to the last generation won't fail.